pkg/deploy/api/types.go

```diff
@@ -445,6 +445,18 @@ type DeploymentConfigRollbackSpec struct {
 	IncludeStrategy bool
 }
 
+// DeploymentRequest is a request to a deployment config for a new deployment.
+type DeploymentRequest struct {
+	unversioned.TypeMeta
+	// Name of the deployment config for requesting a new deployment.
+	Name string
+	// Latest will update the deployment config with the latest state from all triggers.
+	Latest bool
+	// Force will try to force a new deployment to run. It will not work for a paused
```

Comment:
If you try to Force a Paused deployment we should return an error to the client (probably invalid).    Update the docs to describe what happens in the v1 api godoc.